content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -1396,6 +1396,92 @@ curl -X POST \
 <Section>
 <Text>
 
+### List Threads
+
+It returns threads that the current Agent has access to in a given chat.
+
+#### Specifics
+
+|                        |                                                                  |
+| ---------------------- | ---------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/list_threads` |
+| **Required scopes**    | `chats--all:ro` `chats--access:ro`                               |
```

Comment:
what about `chats--my:ro`? I don't remember why `get_chat_threads` doesn't support that

Reply:
actually it supports it in code, seems that it's not documented

I will add `chats--my:ro` to `list-threads`

Reply:
Still missing scope `my`